sideboard/configspec.ini

```diff
@@ -78,8 +78,40 @@ sqlite_dir = string(default="%(root)s/db")
 
 
 [cherrypy]
+checker.check_skipped_app_config = boolean(default=False)
```

Comment:
I moved this line just to put it in alphabetical order. In retrospect that was probably silly, but oh well!